test/tests/selftest/pbs_managers_operators_ext.py

```diff
@@ -50,13 +50,27 @@ def test_managers_unset_setup(self):
         after test setUp run
         """
         runas = ROOT_USER
+        manager_usr_str = str(MGR_USER) + '@*'
         current_usr = pwd.getpwuid(os.getuid())[0]
-        attr = {ATTR_managers: current_usr + '@*'}
-        self.server.expect(SERVER, attrib=attr)
+        current_usr_str = str(current_usr) + '@*'
+        svr_mgr = str(self.server.status(SERVER, 'managers'))
```

Comment:
The code right now only checks that the MGR_USER and the current user are part of managers, it will pass even if managers contains other users in addition to MGR_USER & current user.

How about this instead:
mgr_users = [str(MGR_USER) + '@\*', str(current_usr) + '@\*']
svr_mgr = self.server.status(SERVER, 'managers')[0]['managers'].split(",")
self.assertEquals(set(svr_mgr), set(mgr_users))